turbopack/crates/turbo-rcstr/src/lib.rs

```diff
@@ -337,7 +337,7 @@ pub const fn inline_atom(s: &str) -> Option<RcStr> {
 /// allocates the RcStr inline when possible otherwise uses a `LazyLock` to manage the allocation.
 #[macro_export]
 macro_rules! rcstr {
-    ($s:tt) => {{
+    ($s:expr) => {{
```

Comment:
Maybe this was intended to be part of a separate PR, but this is fine here!

Reply:
ah yes, i think i got confused about branches lol